packages/manager/modules/pci/src/projects/project/storages/databases/components/plans-list/plans-list.controller.js

```diff
@@ -7,6 +7,10 @@ export default class {
     this.$translate = $translate;
   }
 
+  getSortedPlans() {
+    return this.plans?.sort((a, b) => b.compare(a));
```

Comment:
**question:** Where is `compare` function defined?

Reply:
It is defined in the `plan.class.js` file: https://github.com/ovh/manager/blob/a9dc304f4dc738deb8767082d332ee5a815b61ed/packages/manager/modules/pci/src/components/project/storages/databases/plan.class.js#L108